src/databricks/sql/client.py

```diff
@@ -812,6 +815,44 @@ def execute(
 
         return self
 
+    def execute_async(
```

Comment:
Please include the method parameter doc and description as `execute`

Reply:
Added